### PR TITLE
New release v1.1.0

### PR DESCRIPTION
**Pull request by humdingerb**

### CODE REVIEW

File: sources/RuleTab.cpp [hunk 1]
```diff
@@ -359,3 +357,13 @@ RuleTab::MakeEmpty()
 		delete item;
 	}
 }
+
+
+void
+RuleTab::UpdateRules()
+{
+printf("UpdateRules!\n");
```
Comment (waddlesplash) [inline sources/RuleTab.cpp]: Extraneous debugging code

File: sources/RuleRunner.cpp [hunk 2]
```diff
@@ -978,10 +978,10 @@ CommandAction(const BMessage& action, entry_ref& ref)
 
 	int result = system(value.String());
 	if (result) {
-		printf("\tTerminal Command: %s\n\t\tPossible error: "
+		printf("\Shell Command: %s\n\t\tPossible error: "
 			"command returned %d\n", value.String(), result);
 	} else
-		printf("\tTerminal Command: %s\n", value.String());
+		printf("\Shell Command: %s\n", value.String());
```
Comment (waddlesplash) [inline sources/RuleRunner.cpp]: Are you sure you wanted to escape the \S? Didn't you mean to leave the \t there?

File: sources/TestView.cpp [hunk 3]
```diff
@@ -187,7 +188,7 @@ TestView::MessageReceived(BMessage* msg)
 			break;
 		}
 		case MSG_VALUE_CHANGED:
-		{
+		{	// Now done in GetTest, but left for now...
```
Comment (waddlesplash) [inline sources/TestView.cpp]: Why "left here for now"?

**Comment by humdingerb** [comment]
I updated the PR accordingly yesterday. Maybe I should've left a comment here...

**Comment by waddlesplash** [comment]
Thanks! I tagged a release: https://github.com/HaikuArchives/Filer/releases/tag/1.1.0

**Pull request merged**